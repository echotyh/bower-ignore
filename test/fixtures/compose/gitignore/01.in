{
  "bowerDir": "bower_components",
  "in": {
    "bootstrap": {
      "ignore": ["*"],
      "unignore": ["dist/"]
    },
    "font-awesome": {}
  }
}
