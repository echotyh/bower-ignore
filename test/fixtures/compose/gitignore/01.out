{
  "out": [
    "bower_components/bootstrap/*",
    "!bower_components/bootstrap/dist/"
  ]
}
